test/api/v3/integration/groups/PUT-groups.test.js

```diff
@@ -0,0 +1,50 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
+
+describe('Put /group', () => {
+  let groupLeader;
+  let groupName = 'Test Public Guild';
+  let groupType = 'guild';
+
+  beforeEach(async () => {
+    groupLeader = await generateUser({balance: 1});
+  });
+
+  xit('returns an error when a non group leader tries to update', async () => {
```

Comment:
No worries.

And, yes.... But, we have talked about this a fair amount on Github and in slack. You concluded that we need to use the routes to correctly test and to imitate multiple api requests.